client/extensions/woocommerce/woocommerce-services/state/shipping-label/selectors.js

```diff
@@ -260,11 +303,99 @@ const getPackagesErrors = values =>
 		return errors;
 	} );
 
-export const getCustomsErrors = customs => {
-	if ( ! customs.dummyFieldChecked && customs.submitted ) {
-		return { dummyField: 'This must be selected to continue' };
-	}
-	return {};
+export const getCustomsErrors = (
+	packages,
+	customs,
+	destinationCountryCode,
+	destinationCountryName
+) => {
+	const usedProductIds = uniq(
+		flatten( map( packages, pckg => map( pckg.items, 'product_id' ) ) )
+	);
+
+	const valuesByProductId = zipObject( usedProductIds, fill( Array( usedProductIds.length ), 0 ) );
+	forEach( packages, pckg => {
+		forEach(
+			pckg.items,
+			( { quantity, product_id } ) =>
+				( valuesByProductId[ product_id ] += quantity * customs.items[ product_id ].value )
```

Comment:
`values` can be undefined right now if there are no packages in the order. This makes `valuesByProductId` contains `NaN`. The item value is not being sent in `get_selected_packages`:

https://github.com/Automattic/woocommerce-services/blob/ec8db8844c05f33d44d66b9de2077bc48727c839/classes/class-wc-connect-shipping-label.php#L148-L155

I opened an issue here: 

https://github.com/Automattic/woocommerce-services/issues/1465